cdap-test/src/main/java/co/cask/cdap/test/WorkflowManager.java

```diff
@@ -34,12 +35,24 @@
 
   /**
    * Get the list of schedules of the workflow
+   *
+   * @return List of {@link ScheduleDetail}.
+   */
+  List<ScheduleDetail> getProgramSchedules();
+
+  /**
+   * Get the list of schedules of the workflow\
+   *
    * @return List of {@link ScheduleSpecification}.
+   *
+\  * @deprecated since release 4.2. Use {@link #getProgramSchedules()} instead.
```

Comment:
ahh yes this happens on the new macbooks too easily. \ next to [return] is way too sensitive